TERMINOLOGY.md

```diff
@@ -1,53 +1,68 @@
-# Terminology
+# Concepts and terminology
 
 ## Estimators and predictors
-Fairlearn largely follows the [terminology established by scikit-learn](https://scikit-learn.org/stable/developers/contributing.html#different-objects), specifically:
-- Estimator: implements a `fit` method
-- Predictor: implements a `predict` method.
-Unlike [scikit-learn estimators](https://scikit-learn.org/stable/glossary.html#term-estimator) our estimators in fairlearn may not be deterministic. Lots of disparity mitigation techniques require randomized output to meet their objective in expectation. We do provide unrandomized versions of every method as well, but they may not be able to uphold the same theoretical guarantees.
 
-## Randomized predictors
+The fairlearn package largely follows the [terminology established by scikit-learn](https://scikit-learn.org/stable/developers/contributing.html#different-objects), specifically:
+- _Estimators_ implement a `fit` method.
+- _Predictors_ implement a `predict` method.
 
-Rather than providing deterministic output many predictors produced by disparity mitigation techniques in the fairness literature are randomized. As a consequence, it is possible to get different output from the predictor's `predict` method when providing samples with identical features.
 
-## Group Fairness
+**Randomization.** Unlike [scikit-learn estimators](https://scikit-learn.org/stable/glossary.html#term-estimator), estimators in fairlearn can produce randomized predictors. Randomization of predictions is required to satisfy many definitions of fairness. Because of randomization, it is possible to get different outputs from the predictor's `predict` method on identical data. For each of our methods, we provide explicit access to the probability distribution used for randomization.
```

Comment:
Nitpick: this will read "Unlike scikit-learn estimators, estimators in fairlearn" …. I don't think having the same noun on either side of a comma looks good.